lib/ace/ace.js

```diff
@@ -55,6 +55,7 @@ require("./multi_select");
 require("./mode/folding/fold_mode");
 require("./theme/textmate");
 require("./ext/error_marker");
+require("./lib/katex/katex")
```

Comment:
@yanenkoa you're supposed to find the answer to this question :) We don't want to load it early, we want to load it on demand. 
It seems that statusbar demo loads its extension similar to the way we want:https://github.com/ajaxorg/ace/blob/master/demo/statusbar.html